integrations/docker/images/chip-build-android/Dockerfile

```diff
@@ -1,5 +1,5 @@
-ARG VERSION=latest
-FROM connectedhomeip/chip-build:${VERSION}
+ARG chip_build_VERSION=latest
```

Comment:
If 'latest' changes, building the same dockerfile will change between builds even if I do not update either Dockerfile nor 'version'.

I believe that is unintended. How about hard-coding exact versions here instead? That way, builds are always reproducible rather than depending on a label that is more likely to move?

Reply:
or maybe we could mark it to something invalid and then the script has to resolve them correctly or the user can specify via command-line.

Reply:
you're right about latest, which I think is borken anyway.  I can remove the default to latest, requiring that version is given as an argument.  whatcha think?

Reply:
Mandatory seems right. I never want to wrongly update latest.